src/pre_process/m_checker.f90

```diff
@@ -135,9 +135,7 @@ subroutine s_check_inputs()
         elseif (n == 0 .and. p > 0) then
             call s_mpi_abort('Unsupported choice of the combination of '// &
                              'values for n and p. Exiting ...')
-        elseif ((m + 1)*(n + 1)*(p + 1) &
-                < &
-                2**(min(1, m) + min(1, n) + min(1, p))*num_procs) then
+        elseif (nGlobal < 2**(min(1, m) + min(1, n) + min(1, p))*num_procs) then
```

Comment:
how does this evaluate when the right side of `nGlobal < 2**(min(1, m) + min(1, n) + min(1, p))*num_procs` overflows? (since presumably this is what you are checking for?) or am I missing something else. actually, what are we actually checking for?

Reply:
`2**(min(1, m) + min(1, n) + min(1, p))*num_procs` is just `(2^num_dims)*num_procs`, it should never overflow.

Reply:
`nGlobal` replaces `(m+1)*(n+1)*(p+1)` which overflows at `2^31`

Reply:
The other modified check is `(weno_order^num_dims)*num_procs`, which should also never overflow. It would take 17.2m cores with WENO 5 for it to overflow.